test/integration_serial_test.go

```diff
@@ -229,3 +229,27 @@ func TestTerragruntSourceMapEnvArg(t *testing.T) {
 	tgArgs := fmt.Sprintf("terragrunt run-all apply -auto-approve --terragrunt-log-level debug --terragrunt-non-interactive --terragrunt-working-dir %s", tgPath)
 	runTerragrunt(t, tgArgs)
 }
+
+func TestTerragruntSourceMapDebug(t *testing.T) {
+	fixtureSourceMapPath := "fixture-source-map"
+	cleanupTerraformFolder(t, fixtureSourceMapPath)
+	targetPath := "C:\\test\\infrastructure-modules/"
+	copyEnvironmentToPath(t, fixtureSourceMapPath, targetPath)
+	rootPath := filepath.Join(targetPath, fixtureSourceMapPath)
+
+	os.Setenv(
+		"TERRAGRUNT_SOURCE_MAP",
+		strings.Join(
+			[]string{
+				fmt.Sprintf("git::ssh://git@github.com/gruntwork-io/i-dont-exist.git=%s", targetPath),
+				fmt.Sprintf("git::ssh://git@github.com/gruntwork-io/another-dont-exist.git=%s", targetPath),
+			},
+			",",
+		),
+	)
+	tgPath := filepath.Join(rootPath, "multiple-match")
+	tgArgs := fmt.Sprintf("terragrunt run-all apply -auto-approve --terragrunt-log-level debug --terragrunt-non-interactive --terragrunt-working-dir %s", tgPath)
+	fmt.Println(tgArgs)
+	runTerragrunt(t, tgArgs)
+
+}
```

Comment:
NIT: consider adding this to `integration_windows_test.go` so it only runs on windows machines.

Reply:
Good call - moved in d1082e9